### DISCUSSION TITLE
No Rush / No Promises - ETA?

### DISCUSSION THREAD

**Discussion by MadDog443** [lehjr/MachineMusePowersuits#74] [Q&A]
1.18.x - ?
1.19.x - ?

**Comment by lehjr** [comment]
a 1.19.3 port is mostly done and I'm in the process of testing it before the first alpha. It still needs some of the recipes redone. However, 1.19.3 seems to be less popular than 1.19.2, so I'm not sure if it's worth backporting once finished or not.
